setup/bindir/cloud-setup-databases.in

```diff
@@ -576,7 +580,11 @@ for example:
         self.parser.add_option("-j", "--encryption-jar-path", action="store", dest="encryptionJarPath", help="The path to the jasypt library to be used to encrypt the values in db.properties")
         self.parser.add_option("-n", "--encryption-key-file", action="store", dest="encryptionKeyFile", help="The name of the file in which encryption key to be generated")
         self.parser.add_option("-b", "--mysql-bin-path", action="store", dest="mysqlbinpath", help="The mysql installed bin path")
-
+        self.parser.add_option("-s", "--create-schema-only", action="store_true", dest="createschema", default=False,
+                          help="Creates the db schema without having to pass root credentials - "\
```

Comment:
any specific reason why we don't need to provide root credentials here ? I agree if we might not need it to create schema but in general, for running these kind of scripts I think it is good to ask the credentials. Correct me if anything else needs to consider. @Spaceman1984

Reply:
This will allow flexibility in the sense that if 2 people in an organization are given two different roles. One person can create the databases and another can fire scripts on it without having the root credentials.